Core/src/org/sleuthkit/autopsy/logicalimager/configuration/ConfigWizardPanel1.java

```diff
@@ -32,7 +32,7 @@
 /**
  * Configuration Wizard Panel 1
  */
-public class ConfigWizardPanel1 implements WizardDescriptor.ValidatingPanel<WizardDescriptor> {
+class ConfigWizardPanel1 implements WizardDescriptor.ValidatingPanel<WizardDescriptor> {
```

Comment:
fireChangeEvent should be able to be a package scope instead of protected scope because it is final and the class is not public